src/components/DashboardPage/index.jsx

```diff
@@ -0,0 +1,46 @@
+import "./dashboardPage.css";
+import Header from "../Header/Header";
+import React from 'react';
+import { Card, Col, Row } from 'antd';
+import { Button } from 'antd';
+const DashboardPage = () => {
+    return  (
+        <div>
+            <Header />
```

Comment:
This header will be in the parent component already common to all page components, so you can remove it from here. Then the outermost div can also be removed, so you just have one div (with class 'dashboard', see below) and the `Row` is inside it